canonical-web-and-design/hiring.canonical.com
# Change written submission to written interview

## Done
Change written submission to written interview

## QA
- Run server: `dotrun`
- Go to `http://0.0.0.0:8501/stages/` link
- Select "Written Interview" stage
 - Verify graph is displayed correctly
 - Verify all "written submission" labels to "written interview"

## Issue
Fixes #147
